Fixed print issue for TensorTypeId

lauragustafson opened the pull request:
Summary:
Fixed printing issue for TensorTypeID. It used to print a hex of the ID, e.g.
   /x1
Now it prints the ID as a string, e.g.
  1

Differential Revision: D10224026

ezyang reviewed (approved):
Ooooh, it's because 'int8_t' has a different `<<` implementation than `int64_t`! Nicely done.

lauragustafson committed:
Fixed print issue for TensorTypeId (#12402)

Summary:
Fixed printing issue for TensorTypeID. It used to print a hex of the ID, e.g.
   /x1
Now it prints the ID as a string, e.g.
  1
Pull Request resolved: https://github.com/pytorch/pytorch/pull/12402

Reviewed By: ezyang

Differential Revision: D10224026

Pulled By: ezyang

fbshipit-source-id: f54f0770613166c594599e8df995df5c441f50b8
